test/ctia/bundle/routes_test.clj

```diff
@@ -505,7 +528,20 @@
 
        (is (= 2 (count (:sightings bundle-get-res-4))))
        (is (nil? (:indicators bundle-get-res-4)))
-       (is (= 402 (count (:relationships bundle-get-res-4))))))))
+       (is (= 402 (count (:relationships bundle-get-res-4))))
+
+       (is (= (count (:sightings bundle-get-res-3))
+              (count (:sightings bundle-get-res-5))))
+       (is (= (count (:relationships bundle-get-res-3))
+              (count (:relationships bundle-get-res-5))))
+       (is (= (count (:indicators bundle-get-res-3))
+              (count (:indicators bundle-get-res-5))))
+
+       (is (empty? (:relationships bundle-from-source)))
```

Comment:
I think that you should check that the related entities (relationship, sighting) is returned if you export a bundle with one of the Indicator ID.